README.md

```diff
@@ -117,22 +117,22 @@ nix-env -iA cocogitto
 
 ## Conventional commits
 
-To create conventional commits you can use the `coco` binary. It has subcommands for the default `fix` and`feat`
+To create conventional commits you can use the `cog commit` utility. It has subcommands for the default `fix` and `feat`
```

Comment:
If this deprecation has already happened I suggest merging the documentation as soon as possible so newcomers start using the long term solution from the getgo. Also document that the old name was deprecated. Then the actual removal later won't come as *quite* as much of a shock to the system.

Reply:
I suspect this is more relevant to cocogitto/website#7 since there is more impact there

Reply:
I'll take care of this asap, although I am not sure how I should document the deprecation in the documentation website ? Shall I had a migration guide page or just mention it the the relevant section ?

Reply:
Migration guide seems like an overkill, a short notice in bold about `coco` having been deprecated should probably be enough

Reply:
I added a quick PR #183 with a depreaction notice in README